cADFS.psm1

```diff
@@ -940,31 +940,33 @@ class cADFSNode {
     [System.Boolean] Test() {
         # Assume compliance by default
         $Compliant = $true;
-
+        $SyncProperties = $null
```

Comment:
Doesn't need to be set to $null. If the command fails to return anything then it'll be $null.